autogpts/autogpt/.env.template

```diff
@@ -235,3 +235,12 @@ OPENAI_API_KEY=your-openai-api-key
 # AP_SERVER_PORT=8000
 # # AP_SERVER_DB_URL - Specifies what connection url the agent protocol database will connect to (Default: Internal SQLite)
 # AP_SERVER_DB_URL=sqlite:///data/ap_server.db
+
+
+################################################################################
+### White/Black Listing URLS
+################################################################################
+##WEB_POLICY - Specifies whether the url list is black list(0) or a whilte list(1)
+#WEB_POLICY=0
+##URL_LIST - Comma separated URLs to be black or whitelisted, sets according to WEB_POLICY value
+#URL_LIST=https://www.google.com,http://www.google.com
```

Comment:
The only thing I don't like about putting it here, is that these lists might get pretty long. That's why in my version I was attempting to parse files.